akka-actor-tests/src/test/scala/akka/io/dns/DnsSettingsSpec.scala

```diff
@@ -120,11 +134,41 @@ class DnsSettingsSpec extends AkkaSpec {
           resolve-timeout = 1s
           search-domains = "example.com"
           ndots = 5
+          positive-ttl = forever
+          negative-ttl = never
         """))
 
       dnsSettings.NDots shouldEqual 5
     }
 
+    "parse ttl" in {
+      val dnsSettingsNeverForever = new DnsSettings(
+        eas,
+        ConfigFactory.parseString("""
+          nameservers = "default"
+          resolve-timeout = 1s
+          search-domains = []
+          ndots = 1
+          positive-ttl = forever
+          negative-ttl = never
+        """))
+
+      dnsSettingsNeverForever.positiveCachePolicy shouldEqual CachePolicy.Forever
+      dnsSettingsNeverForever.negativeCachePolicy shouldEqual CachePolicy.Never
+
+      val dnsSettingsDuration = new DnsSettings(
+        eas,
+        ConfigFactory.parseString("""
+          nameservers = "default"
+          resolve-timeout = 1s
+          search-domains = []
+          ndots = 1
+          positive-ttl = 10 s
+          negative-ttl = 10 d
```

Comment:
As we use inline config we have to add this line in all other config, otherwise we'll got an error about missing settings. 
If remove some other property, for example `search-domains`, it will fail with the same error. So I did according to general approach in this component.

Reply:
I'm purely talking about the two TTL related ones, which are default in https://github.com/akka/akka/pull/27589/files#diff-618d4a56737cdd72d358885898017a3dR1062-R1063. Removing those unless overriding fails for you? Maybe the test doesn't pickup the defaults but it should.